pdsfile/pdsfile.py

```
@@ -5553,10 +5558,14 @@ def modified_sort_key(basename):
             if matchobj is not None:
                 splits = matchobj.groups()
-                parts = [splits[0],
-                         -cls.version_info(splits[1])[0],
-                         matchobj.group(2),
-                         matchobj.group(3)]
+                # For PDS4, we capture bundle set + version, so two groups
+                if len(splits) == 2:
+                    parts = [splits[0], -cls.version_info(splits[1])[0], '', '']
+                else:
+                    parts = [splits[0],
+                            -cls.version_info(splits[1])[0],
```

Flake8 (7.3.0)

[error] continuation line under-indented for visual indent

(E128)